e2e/src/app.e2e-spec.ts

```diff
@@ -8,6 +10,57 @@ describe('workspace-project App', () => {
     page = new AppPage();
   });
 
+  it('check the heading title', () => {
```

Comment:
In this test format, you are supposed to write in them in the form
```
describe('my application', () => {
...
it('shows the heading title')
..
So that it forms the sentence "my application shows the heading title"